config/locales/en.yml

```diff
@@ -9,6 +9,12 @@ en:
     type:
       return: Changes requested
       begin_deposit: Deposited
+      rejected: Changes requested
+      pending_approval: Waiting for review
+      first_draft: First draft
+      version_draft: Version draft
+      depositing: Deposited
```

Comment:
Do we need both `deposited` and `depositing`?  Do we need `rejected` and `return`?

Reply:
I wish we didn't need them all but my testing found we did.  Perhaps the forthcoming "normalization of event/state display code text" PR will address?   Perhaps wiping the databases will address?  Dunno.  I didn't do the coding that required these -- I was only a consumer stymied by their absence.

Reply:
I pulled this branch down and removed `return` and `depositing` and there were no test failures.  Perhaps this code needs coverage?

Reply:
@jcoyne @ndushay I would like to issue a follow-up PR that tweaks the events attached to state transitions and how we label them. Maybe I could take this on?

Reply:
FYI, I think "depositing" is what you get locally when you hit deposit and the thing spins forever unless you manually give it a druid;  having an extra translation for that isn't a travesty, IMO.